src/Symfony/Component/Routing/Loader/AnnotationClassLoader.php

```diff
@@ -119,49 +124,137 @@ public function load($class, $type = null)
             }
         }
 
+        /** @var $annot RouteAnnotation */
         if (0 === $collection->count() && $class->hasMethod('__invoke') && $annot = $this->reader->getClassAnnotation($class, $this->routeAnnotationClass)) {
             $globals['path'] = '';
             $globals['name'] = '';
+            $globals['locales'] = array();
             $this->addRoute($collection, $annot, $globals, $class, $class->getMethod('__invoke'));
         }
 
         return $collection;
     }
 
-    protected function addRoute(RouteCollection $collection, $annot, $globals, \ReflectionClass $class, \ReflectionMethod $method)
+    /**
+     * @param RouteCollection  $collection
```

Comment:
we removed what we considered "useless" docblocks previously
Id' suggest to do a partial docblock here, listing only $annot and $globals (that's supported : )

Reply:
That makes sense :)